test/index.ts

```diff
@@ -389,4 +389,22 @@ describe('teeny', () => {
       }
     );
   });
+
+  it('should respect followRedirect option being disabled', done => {
```

Comment:
add a test for the positive case of default `followRedirect: true`, this will end up testing `node-fetch`'s default, but it's a good idea to ensure `teeny-request`'s public contract.